client/blocks/app-promo/index.jsx

```diff
@@ -148,29 +153,36 @@ export class AppPromo extends Component {
 		const { translate } = this.props;
 		const buttons = [ { action: 'cancel', label: translate( 'OK' ) } ];
 
+		const message = displayJetpackAppBranding ? (
+			<p className="app-promo__paragraph">
+				<a href="https://wp.com/app" className="app-promo__jetpack-app-link">
+					Get the Jetpack app
+				</a>{ ' ' }
+				{ translate( 'to use Reader anywhere, any time.' ) }
```

Comment:
Splitting strings like this can be problematic in some languages. "Get the Jetpack app to use Reader ..." should be a single string, and we can use the `translate()` componentization features to achieve that - see this for example:

https://github.com/Automattic/wp-calypso/blob/f2e6f88ae65c8571bbac21b284c29b48ec987598/apps/notifications/src/panel/templates/body.jsx#L134